### ISSUE TITLE
Add Back Augmentations 

### ISSUE THREAD

**Issue by leonardoaraujosantos** [leonardoaraujosantos/ChangeNet#2]
### References
* [Pytorch Issue Message](https://github.com/pytorch/vision/issues/9)
* [Pytorch Discussion](https://discuss.pytorch.org/t/torchvision-transfors-how-to-perform-identical-transform-on-both-image-and-target/10606/6)
* [Pytorch Discussion 2](https://discuss.pytorch.org/t/joint-transforms-for-torchvision-datasets-vocsegmentation/46779)

**Comment by leonardoaraujosantos** [comment]
I've added back some augmentations, Including the swap (reference, test) which is confirmed that does NOT help on the current architecture, also unfreezing the Resnet branches gave 12%+ accuracy boost on the validation set.

<img width="369" alt="image" src="https://user-images.githubusercontent.com/898383/65860917-314e6700-e363-11e9-8318-907be2bac852.png">
### Labels
* Orange: Baseline with Resnets frozen
*  Magenta: Augmentations with Resnet unfrozen
* Green: Swap with Augmentations and Resnet unfrozen

TODO: Still add the flip on: (Reference, Test, and label) with same probability
